BGAnimations/ScreenEditMenu underlay/LastSeenSong.lua

```diff
@@ -0,0 +1,111 @@
+-- Retrieve ThemePref values and attempt to set the EditMenu
+-- to the "last seen" song and stepchart for quicker navigation.
+-- This can be handy when a player is working on the same stepchart
+-- over the course of multiple days/SM5 restarts.
+--
+-- These steps (retreiving strings from ThemePrefs and setting GAMESTATE)
+-- can be performed immediately when this file is loaded.  None of it requires
+-- Actors.
+--
+-- The 2nd half of this (writing the LastSeen string values to ThemePrefs)
+-- is handled lower in this file, and hooks into the screen's OffCommand.
+-- It will be run when the player chooses a song and stepchart and successfully
+-- proceeds to ScreenEdit, but not when the player backs out of ScreenEditMenu
+-- (Escape key or mapped BACK button).
+-- ----------------------------------------------------------------
+
+local song_str      = ThemePrefs.Get("EditModeLastSeenSong")
+local diff_str      = ThemePrefs.Get("EditModeLastSeenDifficulty")
```

Comment:
How does this handle files with multiple edits? Does it simply pick the first one?

Reply:
> Does it simply pick the first one?
Yes.